test/testlib/nuodb_database_utilities.go

```diff
@@ -92,6 +92,28 @@ func EnsureDatabaseNotRunning(t *testing.T, adminPod string, opt ExtractedOption
 	k8s.RunKubectl(t, kubectlOptions, "exec", adminPod, "--", "nuocmd", "check", "database", "--db-name", opt.DbName, "--num-processes", "0", "--timeout", "30")
 }
 
+func RestartDatabasePods(t *testing.T, namespaceName string, helmChartReleaseName string, options *helm.Options) {
+	opt := GetExtractedOptions(options)
+	hcSmPodNameTemplate := fmt.Sprintf("sm-%s-nuodb-%s-%s-hotcopy", helmChartReleaseName, opt.ClusterName, opt.DbName)
+	smPodNameTemplate := fmt.Sprintf("sm-%s-nuodb-%s-%s", helmChartReleaseName, opt.ClusterName, opt.DbName)
+	tePodNameTemplate := fmt.Sprintf("te-%s-nuodb-%s-%s", helmChartReleaseName, opt.ClusterName, opt.DbName)
+	var toDelete []string
+	tes := GetPodNames(t, namespaceName, tePodNameTemplate)
+	require.Equal(t, opt.NrTePods, len(tes), "Unexpected number of TE Pods")
+	toDelete = append(toDelete, tes...)
```

Comment:
Was not familiar with the `...` syntax. I guess this expands an array when invoking a variadic function.

Reply:
Yes. I was reminded of this by the IDE. :)